conf/application.conf

```diff
@@ -49,6 +49,8 @@ tracking-consent-frontend {
   gtm.container = "a"
 }
 
+emailAuthEnabled = true
```

Comment:
I think this name is misleading. May be we can call this `emailAuth.generateTestCode=true/false`